client/js/models/archive.js

```diff
@@ -205,7 +209,7 @@ module.exports = {
       }
       if (!archive) {
         archive = drive.createArchive(key)
-        sw = swarm(archive)
+        sw = swarm(archive, {signalhub: state.signalhubs})
```

Comment:
yes, the dat.land signahub is in the initialState and we don't currently have any action to change it.  It can only be overridden by env on start time.

I've seen sporadic instability, and that's why I want the test here to use a local one.  For the live ones, we should use some kind of monitor service to track their availabilities and performance.